src/Ledger/Gov.lagda

```diff
@@ -95,6 +95,18 @@ govActionPriority (ChangePParams _)        = 4
 govActionPriority (TreasuryWdrl _)         = 5
 govActionPriority Info                     = 6
 
+_gaPrRel_ : ℕ → ℕ → Type
```

Comment:
I don't like this name. It's not descriptive and you already need to know what it is to have an idea what the name could mean. The name either has to be some long self-explanatory name or it could just be a variation on `≈` together with an explanation in the prose (which I prefer).

Reply:
I will rename the relation `_∼_` and will add the following explanation to the prose:

"Additionally, \govActionPriority assigns a priority to the various governance action types.  This is useful for ordering lists of governance actions as well as grouping governance actions according to its constructor (\NoConfidence, \UpdateCommittee, \NewConstitution, etc.).  In particular, the relations \AgdaFunction{_∼_} and \AgdaFunction{_≈_} defined in Figure~\ref{defs:enactable} are used for determining whether two actions are of the same ``kind'' in the following sense: either the actions arise from the same constructor, or one action is a \NoConfidence and the other is an \UpdateCommittee action."